IFC4x3/Properties/d/DriveDirection_2PE$H34I5DPu1AAA2NFe$2/DocProperty.xml

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Looking at the values I would propose the following change to this property:
- remove the enum
- change property type to `IfcBoolean`
- rename the property to `IsDriveDirectedAlongChainage`

Reply:
This comment should be answered by @JontyDunn

Reply:
Property changed to IsDriveDirectedAlongChainage : IfcBoolean.